src/OpenIDConnect.jl

```diff
@@ -47,16 +47,10 @@ struct OIDCCtx
         end
 
         if cacrt !== nothing
-            isa(cacrt, String) && (cacrt = MbedTLS.crt_parse(base64decode(cacrt)))
-            conf = MbedTLS.SSLConfig()
-            MbedTLS.config_defaults!(conf)
-            entropy = MbedTLS.Entropy()
-            rng = MbedTLS.CtrDrbg()
-            MbedTLS.seed!(rng, entropy)
-            MbedTLS.rng!(conf, rng)
-            MbedTLS.authmode!(conf, ((verify === nothing) || verify) ? MbedTLS.MBEDTLS_SSL_VERIFY_REQUIRED : MbedTLS.MBEDTLS_SSL_VERIFY_NONE)
+            isa(cacrt, String) && (cacrt = MbedTLS.crt_parse_file(cacrt))
```

Comment:
We could probably retain the old behavior as well by checking if the `cacrt` string is indeed a readable file. E.g.:
```julia
if isa(cacrt, String)
    cacrt = isfile(cacrt) ? MbedTLS.crt_parse_file(cacrt) : MbedTLS.crt_parse(base64decode(cacrt))
end
```

Reply:
I had the same idea :-)

Reply:
But I'd tend to just pass `cacert` to `MbedTLS.crt_parse()`, it handles stripping of  `-- BEGIN/END CERTIFICATE --` lines and can handle more than a single certificate.
What do you think?

Reply:
Yes, that should be fine too